src/app/pages/ProfileView/ProfileView.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useEffect } from 'react'
+import { BiChevronsRight } from 'react-icons/bi'
+import { useNavigate, useParams } from 'react-router-dom'
+import styled from 'styled-components'
+import Button from '../../components/Button/Button'
+import Logo from '../../components/Logo/Logo'
+import useFetch from '../../hooks/useFetch'
+import type { User } from '../../types/types'
+
+export default function ProfileView(): JSX.Element {
+  const { username } = useParams()
+  const nav = useNavigate()
+
+  const [user, fetchUser] = useFetch<User>('/api/users', { username })
+
+  useEffect(() => {
+    fetchUser('GET')
+  }, [])
+
+  return (
+    <ProfileContainer>
+      <Navbar>
+        <Logo />
+        <Button onClick={() => console.log('logout')}>logout</Button>
+      </Navbar>
+      {user && (
+        <Content>
+          <Greeting>Welcome back {user.username}</Greeting>
+          <BoardsContainer>
+            <h2>Your Boards</h2>
+            <BoardList>
+              {user.boards.map(board => (
+                <Card onClick={() => nav(`/boards/${board}`)}>
+                  <BoardName>{board}</BoardName>
+                  <BiChevronsRight size="32px" />
+                </Card>
+              ))}
+            </BoardList>
+            <Button>Create new Board</Button>
+          </BoardsContainer>
+        </Content>
+      )}
+    </ProfileContainer>
+  )
+}
+const ProfileContainer = styled.div`
+  display: grid;
+  height: 100vh;
+  grid-template-rows: auto 1fr;
+`
+const Navbar = styled.nav`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 15px;
+  z-index: 10;
+`
+const Card = styled.li`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px;
+  background-color: var(--c-gray-50);
```

Comment:
I really appreciate that you have set this variable, i need to integrate that too!